### DISCUSSION TITLE
[Kavita] SFW/NSFW mode

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2536] [Ideas]
### Idea Description

SFW/NSFW mode
Posted by ホロ
· Apr 2022
· 66 Combined Upvotes between:
https://feats.kavitareader.com/posts/112/sfw-nsfw-mode
https://feats.kavitareader.com/posts/37/blurring-cover-image-of-inappropriate-nsfw-content

Description

There's already a request here to blur the cover on NSFW covers but I'm thinking it would be nice to have a SFW/NSFW toggle as well. What this means is that when SFW mode is on all NSFW tagged content is inaccessible. There could also be other views where its only NSFW content appears as well as one where nothing is hidden. The NSFW tag could be applied per item, per library, or both.

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by ghost** [comment]
Basically 2 Feat Requests, but you kinda merged it into something completly else.
While i like that you merged the requests to GH, i would have preffered if you kept them seperate and 1:1 of the Feats...
Anyways i created [my own Idea Request](https://github.com/Kareadita/Kavita/discussions/2766), resembling the original Request.

**Reply by scare376** [reply]
They were 2 different requests I just referenced the blur request when I made mine. The other one just didn't get ported over for whatever reason. Oh I see the votes got merged.
